project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,181 @@
+---
+breaks: False
+---
+# Newsletter | 29 January 2020
+
+*Date sent: 01 February 2020*
+
+Hello, 
+
+We're already one month into 2021!
+We hope your year has had a good beginning. 
+For those struggling with these challenging times, we hope that the citizen science community can offer your support and connection. 
+
+Email Georgia at [gaitkenhead@turing.ac.uk](mailto:gaitkenhead@turing.ac.uk) to let us know if we can help you with involvement, and do feel free to share this newsletter to anyone who you think would be interested in joining us. 
+
+They can sign up by following this link: [https://tinyletter.com/AutisticaTuringCitizenScience](https://tinyletter.com/AutisticaTuringCitizenScience).
+
+Best wishes for everyone!
```

Comment:
possibly rephrase to "best wishes to everyone" or "Sending well wishes to all of you"